pre_install_report/library/pre_install_check_permissions.py

```diff
@@ -394,23 +401,59 @@ def check_sample_service(self):
         """
         Deploy Kubernetes Service for hello-world appliction in specified namespace and set the
         permissions status in the namespace_admin_permission_data dict object
-
-
         """
 
         rc = self.utils.deploy_manifest_file(viya_constants.KUBECTL_APPLY,
                                              'helloworld-svc.yaml')
         self._set_results_namespace_admin(viya_constants.PERM_SERVICE, rc)
 
+    def get_server_gitVersion(self):
+        """
+        Retrieve the Kubernetes servervrsion and validate the gitVersion
+        """
+        try:
+            versions: Dict = self.utils.get_k8s_version()
+            serverversion = versions.get('serverVersion')
+            gitVersion = serverversion.get('gitVersion')
```

Comment:
should be named server_version and git_version

Reply:
Completed